libs/basics/src/map_object.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Takes a dictionary and returns a dictionary with the same keys but values
+ * remapped by a provided function.
+ */
+export function mapObject<T, U>(
```

Comment:
Could use `_.mapValues()` instead: https://lodash.com/docs/4.17.15#mapValues

(That is, unless we have some guiding principles on the team around not using lodash - I did notice we don't have too many usage instances of it in the codebase)

Reply:
It's ambiguous, but I think we have an unspecified preference for not using `lodash` or other easily replicable dependencies 🤷